applications/GeoMechanicsApplication/custom_constitutive/mohr_coulomb_with_tension_cutoff.cpp

```diff
@@ -317,4 +317,25 @@ void MohrCoulombWithTensionCutOff::FinalizeMaterialResponseCauchy(ConstitutiveLa
     mStressVectorFinalized = mStressVector;
 }
 
+void MohrCoulombWithTensionCutOff::save(Serializer& rSerializer) const
+{
+    rSerializer.save("ConstitutiveLawDimension", mpConstitutiveDimension);
+    rSerializer.save("StressVector", mStressVector);
+    rSerializer.save("StressVectorFinalized", mStressVectorFinalized);
+    rSerializer.save("StrainVectorFinalized", mStrainVectorFinalized);
+    rSerializer.save("CoulombYieldSurface", mCoulombYieldSurface);
+    rSerializer.save("TensionCutOff", mTensionCutOff);
+    rSerializer.save("mIsModelInitialized", mIsModelInitialized);
```

Comment:
To make it consistent with the other members, I'd change this to:
```suggestion
    rSerializer.save("IsModelInitialized", mIsModelInitialized);
```

Reply:
fixed